android/src/main/java/com/oblador/keychain/cipherStorage/CipherStorageKeystoreAesCbc.java

```diff
@@ -215,6 +218,32 @@ protected Key generateKey(@NonNull final KeyGenParameterSpec spec) throws Genera
 
     return generator.generateKey();
   }
+
+  /** Decrypt provided bytes to a string. */
+  @NonNull
+  @Override
+  protected String decryptBytes(@NonNull final Key key, @NonNull final byte[] bytes,
+                                @Nullable final DecryptBytesHandler handler)
+    throws GeneralSecurityException, IOException {
+    final Cipher cipher = getCachedInstance();
+
+    // decrypt the bytes using cipher.doFinal(). Using a CipherInputStream for decryption has historically led to issues
+    // on the Pixel family of devices.
+    // see https://github.com/oblador/react-native-keychain/issues/383
+    try {
+      // read the initialization vector from bytes array
+      if (null != handler) {
+        handler.initialize(cipher, key, bytes);
+      }
```

Comment:
```suggestion
      final IvParameterSpec iv = IV.readIv(bytes);
      cipher.init(Cipher.DECRYPT_MODE, key, iv);
```

In order to not have to make the handler interface generic.

I realise that this will make the implementation deviate a bit from the other `CipherStorage` classes, but probably best to keep this workaround as contained as possible.